src/data/hero.ts

```diff
@@ -42,105 +31,30 @@ const getBandProperties = (
-            const patrolSlot = getPatrolSlot($, e.instance);
-            if (patrolSlot === "SCAVENGER") {
-              return [
-                {
-                  type: "GIVE_GOLD",
-                  sourceCard: I.card,
-                  sourceInstance: I.id,
-                  player: valueParam("PLAYER", I.controller),
-                  amount: constantParam(1),
-                },
-              ];
-            } else if (patrolSlot === "TECHNICIAN") {
-              return [
-                {
-                  type: "DRAW",
-                  sourceCard: I.card,
-                  sourceInstance: I.id,
-                  player: valueParam("PLAYER", I.controller),
-                  amount: constantParam(1),
-                },
-              ];
-            }
-
-            return [];
-          }),
-        ],
-      };
-    case 2:
-      return {
-        triggeredAbilities: [
-          trigger("MAX_BAND", ($, I) => {
-            if ($.state.activePlayer !== I.controller) {
-              return [];
-            }
-
-            const opponents = Array.from(
-              getOpponents($, $.getPlayer(I.controller)),
-            );
-            const opponentsWithEmptySlots =
-              opponents.filter(P => hasEmptyPatrolSlot(P));
-
-            if (opponentsWithEmptySlots.length === 0) {
-              // The enemy patrol zone is full, so we just do damage to a
-              // patroller without moving it
-              return [
-                {
-                  ...effectBase(id, I, "DAMAGE"),
-                  target: queryParam("INSTANCE", {
-                    patrolling: true,
-                  }),
-                  amount: constantParam(1),
-                },
-              ];
-            }
-
-            if (opponentsWithEmptySlots.length > 1) {
-              // This is difficult to support atm, since we wouldn't know which
-              // patrol slots are valid until we know which player was chosen.
-              throw new Error("shove unsupported vs multiple opponents");
-            }
-            const emptySlots = getEmptyPatrolSlots(opponentsWithEmptySlots[0]);
-
-            // Move a patroller to an empty slot, then deal 1 damage to it
-            return [
-              {
-                ...effectBase(id, I, "MOVE_TO_SLOT"),
-                target: queryParam("INSTANCE", {
-                  patrolling: true,
-                  player: opponentsWithEmptySlots[0].id,
-                }),
-                slot: queryParam("PATROL_SLOT", emptySlots),
-                chainedEffects: [
-                  {
-                    type: "DAMAGE",
-                    target: inheritParam("INSTANCE", "target"),
-                    amount: constantParam(1),
-                  },
-                ],
-              },
-            ];
-          }),
-        ],
-      };
-    default:
-      return {};
-    }
+    return getHeroBand(id, band, CaptainZane);
```

Comment:
Good idea to move this into separate files. We may end up wanting to do this for units/buildings/etc in the future? In which case it might be worth putting a bit of thought into how we want this to look (file for each card with significant text is probably fine, but we could also do a file per spec or per color or sth)

Reply:
I think heroes deserve their own files, for sure. A file per spec might be an acceptable division after that, with starter cards in their own file as well, perhaps? Maybe heroes could be in their spec files, too. A file per card in general is probably overkill.